data-modeling.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+title: Data Modeling
+nav_order: 4
+has_toc: true
+---
+
+# Data Modeling
+
+With Dojo's Data Modeling tool, users can load features from NetCDF files into a visual graphing flow, perform actions on those features, and generate new datasets from the results.
+
+## Selecting NetCDFs
+
+When you initially land on the Data Modeling page, you'll be prompted to selected registered NetCDFs. If you have a NetCDF that you'd like to use that isn't yet registered in Dojo, follow the steps in [Data Registration](./data-registration) to register it. It will then appear in the list of NetCDF datasets.
+
+You can select as many datasets as you'd like. Once you've selected all your datasets, click `USE THESE DATASETS` on the bottom of the page to move to the next step.
+
+![Select NetCDFs](imgs/data-modeling-select-netcdfs.png)
+
+## Building a graph
+
+On the second step, you now have to start building out your graph. Drag and drop the different nodes referenced below from the list of nodes on the right side of the screen onto the dotted surface that covers the majority of the screen. Click on the small circles at the bottom/top of each node, then move your mouse (with the line from the circle) to the circle at the bottom/top of the node you'd like to connect your selected node to. By adding nodes, described below, and connecting them together, you'll create a graph that will allow you to process data from multiple features into new dataset outputs.
+
+### Nodes:
+
+![Node List](imgs/data-modeling-node-list.png)
+
+#### Load Node
+
+![Load Node](imgs/data-modeling-load-node.png)
+This starts with selecting Load Nodes. Each Load Node represents one feature from a dataset. You select this feature with the "Data Source" dropdown at the top of the node. You'll see features listed per dataset in this dropdown.
+
+Once you've selected a feature for a Load Node, you can choose to make it the Geo Resolution and/or the Time Resolution for your new dataset. You can only select each of these once in the entire graph, either in the Load Nodes or in the Time/Geo Resolution selectors in the sidebar.
+
+You also need to choose a Geo and Time Aggregation function for each Load Node.
```

Comment:
```suggestion
Additionally, you must specify what type of regridding method will be used if the data in the load node needs to be regridded. These are specified with the Geo and Time Aggregation function dropdowns for each Load Node:
- `conserve` - maintain the total sum of the data before and after (e.g. for regridding population)
- `min` - take the minimum value from each bucket during regridding
- `max` - take the maximum value from each bucket during regridding
- `mean` - take the mean value from each bucket during regridding. **People should use interp_or_mean instead of this one, since it doesn't handle well when increasing the resolution of the data**
- `median` - take the median value from each bucket during regridding
- `mode`  - take the mode of each bucket during regridding. **People should use nearest_or_mode instead of this one, since it doesn't handle well when increasing the resolution of the data**
- `interp_or_mean`  - if increasing resolution, interpolate the data. If decreasing resolution, take the mean of each bucket
- `nearest_or_mode`  - if increasing resolution, take the nearest data. if decreasing resolution, take the mode of each bucket
```